core/os/memory.cpp

```diff
@@ -43,41 +44,36 @@ void *operator new(size_t p_size, void *(*p_allocfunc)(size_t p_size)) {
 	return p_allocfunc(p_size);
 }
 
-#include <stdio.h>
-
 #ifdef DEBUG_ENABLED
 size_t Memory::mem_usage = 0;
 size_t Memory::max_usage = 0;
+#define PREPAD true
+#else
+#define PREPAD p_pad_align
```

Comment:
That's bogus no? `p_pad_align` is a parameter of `void *Memory::alloc_static(size_t p_bytes, bool p_pad_align)`, I doubt you can use it in the outer scope for a compiler define.